examples/WatchFaces/7_SEG/7_SEG.ino

```diff
@@ -2,7 +2,11 @@
 
 Watchy7SEG watchy;
 
-void setup(){
+void setup() {
+  // City names with spaces require a +
+  // temp units must be either metric or imperial
+  watchy.setWeatherLocation("New+York", "US", "metric");

```

Comment:
Why not use the values from config.h here by default?
```suggestion
  watchy.setWeatherLocation(CITY_NAME, COUNTRY_CODE, TEMP_UNIT);

```

Reply:
> Why not use the values from config.h here by default?

I considered that but it wouldn't be obvious to the user modifying the example INO what the format requirements are.   Here the space/plus and the two-char country code are clear.  Obviously additional comments would help but who reads those :)

Reply:
Wondering if the constants should be deleted entirely, But I guess still useful as a default in case `setWeatherLocation` is not called.

I agree with @adamm, the constants probably shouldn't be used in the example code, there doesn't seem to be any point as these values are used by default anyway.

But for future maybe would be good if we could somehow extract custom config options - maybe a shared *.h file somewhere that isn't in the Watchy library - and not have them hard coded in the code.